website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+- **New**: The dbt Cloud IDE supports signed commits for Git, available for Enterprise plans. You can sign your Git commits when pushing them to the repository to improve identity protection/impersonation and enhance security. Supported Git providers are GitHub and GitLab Refer to [Git commit signing](/docs/cloud/dbt-cloud-ide/git-commit-signing.md) for more information.
```

Comment:
probably just "prevent impersonation" instead of "improve identity protection/impersonation"

Reply:
Updated in this commit: https://github.com/dbt-labs/docs.getdbt.com/pull/6324/commits/73d097b0d7784ef3645b0e715ee364184aaf2471